sklearn/utils/metaestimators.py

```diff
@@ -111,7 +111,10 @@ def __get__(self, obj, owner=None):
                 )
 
         # lambda, but not partial, allows help() to work with update_wrapper
-        out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)  # noqa
+        if obj is None:
+            out = lambda *args, **kwargs: self.fn(*args, **kwargs)  # noqa
+        else:
+            out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)  # noqa
```

Comment:
This line could be moved into the previous block (that starts at line 103) and the unbound lambda can be handled by an `else` close for the previous condition.

But feel free to ignore this comment if you prefer this code organization.

Reply:
Sounds good, fixed!